Makefile.toml

```diff
@@ -155,15 +157,15 @@ category = "Misc"
 description = "List all nodes in the cluster"
 script = '''
 #!/usr/bin/env bash
-tmux list-windows -t risedev | grep -v active | cut -d'(' -f1
+${TMUX} list-windows -t risedev | grep -v active | cut -d'(' -f1
```

Comment:
Since we've already specified a different socket for the server assuming that it's exclusive for the usage of `risedev`, do we still have to specify the session name?

We also kill the entire server on `kill` in this PR.

Reply:
We still have to use `-t session_name` here because we need to put all windows in one session, otherwise each window will have its own session in which case we can't attach to one session and see all nodes.